data/one_player_sleep/functions/loop.mcfunction

```diff
@@ -0,0 +1,23 @@
+##########################################
+# Author: MadCat (youtube.com/MadCatHoG) 
+# Date: Jun 9, 2017
+# Version: 2.0
+# Minecraft Version 1.13
+# Description:
+# One Player Sleep main loop function.
+##########################################
+
+tag @a[nbt={Sleeping:1b}] add isSleeping
+execute if entity @a[tag=isSleeping] run scoreboard players enable @a OPSkickBed 
+scoreboard players add @e[tag=isSleeping] OPSsleep 1
+
+execute if entity @e[tag=isSleeping,scores={OPSsleep=1}] run tellraw @a ["",{"selector":"@s"},{"text":" is now sleeping","color":"blue","clickEvent":{"action":"run_command","value":"/trigger OPSkickBed set 1"},"hoverEvent":{"action":"show_text","value":{"text":"","extra":[{"text":"Click to kick!","color":"aqua"}]}}}]
```

Comment:
It seems this would be better as `@a` instead of `@e`, however this is a direct translation of what was there already